xonsh/built_ins.py

```diff
@@ -599,6 +610,16 @@ def run_subproc(cmds, captured=False):
                 errout = errout.replace('\r\n', '\n')
                 procinfo['stderr'] = errout
 
+    if getattr(prev_proc, 'signal', None):
+        sig, core = prev_proc.signal
+        try:
+            sig_str = SIGNAL_STRINGS[sig]
```

Comment:
Well, I was following the Python "easier to ask forgiveness than permission", but I'll use `get` if you prefer.